pkgs/games/crawl/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, lib, fetchFromGitHub, which, sqlite, lua5_1, perl, zlib, pkgconfig, ncurses
+{ stdenv, lib, fetchFromGitHub, which, sqlite, lua5_1, perl, python37, zlib, pkgconfig, ncurses, curl
```

Comment:
- please use `python3` instead of `python37` to allow for future upgrades
- Why did you add `curl` here? It's not used anywhere below.

Reply:
I went with python37 to be a bit more explicit, but happy to use python3 if it's more idiomatic. Updated.

Oops, curl wasn't supposed to be in there. That was from some earlier testing. Removed.

Thanks!